File: backend/src/Domain/Entities/ProductLine.cs
namespace ProductionMove.Domain.Entities;

public class ProductLineInfo
{
    public string Property { get; set; } = string.Empty;

    public string Value { get; set; } = string.Empty;
}

public class ProductLine : BaseEntity
{
    public string Id { get; set; } = string.Empty;

    public string Name { get; set; } = string.Empty;

    public int WarrantyPeriod { get; set; }

    public virtual ICollection<ProductLineInfo> Describes { get; set; } = new List<ProductLineInfo>();
}

File: frontend/src/data/entities/ProductLine.ts
export interface ProductLineInfo {
    property: string,
    value: string
}

export default interface ProductLine {
    id: string,
    name: string,
    warrantyPeriod: number,             // Thời gian bảo hành của dòng sản phẩm này
    describes: Array<ProductLineInfo>   // Thông số của dòng sản phẩm này
}